domenetjenester/tilbakekreving/src/main/java/no/nav/foreldrepenger/økonomi/tilbakekreving/klient/FptilbakeRestKlient.java

```diff
@@ -1,18 +1,26 @@
 package no.nav.foreldrepenger.økonomi.tilbakekreving.klient;
 
+import java.util.UUID;
+
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.ws.rs.core.UriBuilder;
-import no.nav.foreldrepenger.domene.typer.Saksnummer;
-import no.nav.vedtak.felles.integrasjon.rest.*;
 
-import java.util.UUID;
+import no.nav.foreldrepenger.domene.typer.Saksnummer;
+import no.nav.vedtak.felles.integrasjon.rest.FpApplication;
+import no.nav.vedtak.felles.integrasjon.rest.RestClient;
+import no.nav.vedtak.felles.integrasjon.rest.RestClientConfig;
+import no.nav.vedtak.felles.integrasjon.rest.RestConfig;
+import no.nav.vedtak.felles.integrasjon.rest.RestRequest;
+import no.nav.vedtak.felles.integrasjon.rest.TokenFlow;
 
 @ApplicationScoped
 @RestClientConfig(tokenConfig = TokenFlow.ADAPTIVE, application = FpApplication.FPTILBAKE)
 public class FptilbakeRestKlient {
 
     public static final String FPTILBAKE_HENT_ÅPEN_TILBAKEKREVING = "/api/behandlinger/tilbakekreving/aapen";
 
+    public static final String FPTILBAKE_HENT_ÅPEN_BEHANDLING = "/api/behandlinger/tilbakekreving/aapen-behandling";
```

Comment:
Hva er forskjell på disse 2? åpen behandling og åpen tilbakekreving?

Reply:
Den ene brukes i Simulering for å sjekke om åpen TBK. Har spurt om TBK/rev kan likebehandles - men usikker.
Den andre for fagsakstatus for å sjekke åpen behandling uansett type.